packages/engine/package.json

```diff
@@ -43,6 +43,11 @@
     "@mediapipe/tasks-vision": "0.10.12",
     "@pixiv/three-vrm": "^2.0.6",
     "@tweenjs/tween.js": "^23.1.2",
+    "alea": "1.0.1",
+    "comlink": "4.4.1",
+    "cross-env": "7.0.3",
+    "cross-fetch": "^3.1.5",
+    "detect-gpu": "^5.0.38",
```

Comment:
move detect-gpu back to spatial package, and revert changes to other dependencies (merge error)